Makefile

```diff
@@ -3,7 +3,7 @@ default: all
 all: bdcs-api-server
 
 sandbox:
-	[ -d .cabal-sandbox ] || cabal sandbox init
+	cabal sandbox init
```

Comment:
Isn't this going to always re-initialize the sandbox?

Reply:
From what I can tell only the config file will be updated if a sandbox is already present. There seems to be no ill effect wrt packages already being compiled inside the sandbox.